src/server/repository/public/ddl/getCreatePublicSchemaDDL.ts

```diff
@@ -0,0 +1,162 @@
+export const getCreatePublicSchemaDDL = (schemaName = 'public'): string => {
```

Comment:
Most of this is autogenerated

Some changes to generated file:

Added `if not exists` clases for enums, tables and extensions